tests/std/tests/LWG3018_shared_ptr_function/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\usual_latest_matrix.lst

```

Comment:
Why latest?

Reply:
Because I don't quite understand these matrices...

Reply:
Inspect `usual_latest_matrix.lst` , `usual_matrix.lst` etc one level up.
Usual latest matrix is something that expected to work only starting C++20.
I think that LWG issue resolution may work retroactively.

Reply:
```suggestion
RUNALL_INCLUDE ..\usual_matrix.lst
```